kasa/cli/feature.py

```diff
@@ -122,6 +122,12 @@ async def feature(
 
     feat = dev.features[name]
 
+    if value is None and feat.type is Feature.Type.Action:
+        echo(f"Changing {name} from {feat.value} to {value}")
+        response = await dev.features[name].set_value(value)
```

Comment:
This is an unrelated change that likely deserves its own PR, but a better wording would probably be "Executing action {name}" or something similar?

Reply:
Moved to https://github.com/python-kasa/python-kasa/pull/1264 and updated message